var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -207,6 +207,7 @@ class Hdf5(CMakePackage):
     variant("hl", default=False, description="Enable the high-level library")
     variant("cxx", default=False, description="Enable C++ support")
     variant("map", when="@1.14:", default=False, description="Enable MAP API support")
+    variant("subfiling", when="@1.14:", default=False, description="Enable Subfiling VFD support")
```

Comment:
Where is this variant used in the recipe?

Reply:
Good catch! 
I fixed it. 
Thank you for your review!